cpmpy/tools/explain/mus.py

```diff
@@ -108,6 +111,55 @@ def do_recursion(soft, hard, delta):
     return [dmap[a] for a in core]
 
 
+def optimal_mus(soft, hard=[], weights=1, solver="ortools", hs_solver="ortools"):
+    """
+        Find an optimal MUS according to a linear objective function.
+        Works by iteratively generating correction subsets and computing optimal hitting sets to those enumerated sets.
+        For better performance of the algorithm, use an incemental solver to compute the hitting sets such as Gurobi.
```

Comment:
I guess we should support something like hs_solver=["gurobi", "ortools"] where solverLookup.get(list) would return the first available one? (for a separate pull request clearly... this is just a request for comment on the idea)

Reply:
Hmm, not sure. 
As a novice it's nice to have the likely best solver to be selected but it might unwanted for a more experienced user to have the algo silently default to ortools if there is a problem with the first solver.